src/lib/Components/Bidding/Components/Input.tsx

```diff
@@ -43,6 +50,7 @@ export class Input extends Component<InputProps, InputState> {
         fontSize={3}
         onBlur={() => this.onBlur()}
         onFocus={() => this.onFocus()}
+        ref={component => this.props.createCustomRef && this.props.createCustomRef(this.props.refName, component)}
```

Comment:
This is sort of tricky because we want to define a ref _on_ the parent `BillingAddress` component, but we want it to point to this `TextInput` because it's the only element which has `focus` and `blur` methods. Here, we pass the component up to the `createCustomRef` method on the parent element.